apps/judicial-system/web/src/utils/stepHelper.tsx

```diff
@@ -6,6 +6,8 @@ import {
   Case,
   CaseAppealDecision,
   CaseCustodyRestrictions,
+  CaseGender,
+  Gender,
```

Comment:
CaseGender and Gender are not used in this file. Please remove.

Reply:
Good eye, thanks 🙏